packages/apps-config/src/api/spec/moonbeam.ts

```diff
@@ -1,17 +1,77 @@
 // Copyright 2017-2020 @polkadot/apps-config authors & contributors
-// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier": Apache-2.0
```

Comment:
Must be an accident, right?

Reply:
Lint has got it - https://github.com/polkadot-js/apps/pull/3961/checks?check_run_id=1365083861#step:3:182

Reply:
good catch :)